docs/examples/prototypes.md

```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+Would you like to explore the applications that you could build with Amadeus Self Service APIs? 
+We have prototypes available in [Amadeus for Developers Github](https://github.com/amadeus4dev).
+
+
+Here on this page, we have listed down our prototypes(examples or Demo) based on **use case** for your convenience. 
+
+
+| Use Cases                   | 1A API used                                                                                                | Technology           | Github Link                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Maybe external people don't know 1A is Amadeus, so better to say Amadeus API used?

Reply:
same for Github -> GitHub